src/mame/aristocrat/aristmk7.cpp

```diff
@@ -0,0 +1,235 @@
+     SLA9H - Intel 82945 Memory / Video Controller (northbridge + Calistoga GPU)
+     SL9LF - Intel Celeron-M 440 CPU (1.86GHz / 1M cache / 533 FSB). Clock 266 * 7 = 1862MHz
+  PC82573L - Intel PC82573L Gigabit Ethernet Controller with 25MHz OSC on bottom side of PCB
+   NH82801 - Intel NH82801GBM I/O Controller (southbridge: SATA, USB2.0 etc)
+CY28411ZXC - Cypress (now Infineon) CY28411ZXC Clock Generator with 266MHz output
+CY25823ZXC - Cypress (now Infineon) CY25823ZXC PLL (Phase Lock Loop)
+
+Other parts on bottom: (mostly power-related)
+  ISL6227CAZ - Intersil (now Renesas) ISL6227CAZ Dual PWM Controller (x2)
+       FDS70 - Dual Switching Diode (6 on PCB at various locations)
+       4816B - N-channel Power MOSFET
+ SLB9635TT12 - Infineon TPM1.2 IC with connected 32.768kHz oscillator
+       LM339 - Texas Instruments LM339 Quad Differential Comparator
+      12F509 - Microchip PIC12F509 8-bit Flash Microcontroller marked '28181 V1.2'
+    W83L7866 - Winbond W83L7866 Hardware Monitor IC (connected to PIC12F509)
+ ATMLH00804B - Atmel AT24C02 256-byte EEPROM with custom / obfuscated markings located on bottom
+               near RAM slots
+  ISL6262CRZ - Intersil (now Renesas) ISL6262CRZ Two-Phase Buck Converter
+663674140945 \ ? in silver tin-can enclosed shell. Two of each present on PCB
+663582190951 / and located near ISL6262
+
+Apparently two PCB revisions, one is red coated.
+
+**************************************************************************************************/
+
+
+#include "emu.h"
+#include "cpu/i386/i386.h"
+#include "machine/pci.h"
+
+
+namespace {
+
+class aristmk7_state : public driver_device
+{
+public:
+	aristmk7_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+	{ }
+
+
+	void aristmk7(machine_config &config);
+
+private:
+	required_device<cpu_device> m_maincpu;
+	void aristmk7_io(address_map &map);
+	void aristmk7_map(address_map &map);
+};
+
+
+void aristmk7_state::aristmk7_map(address_map &map)
+{
+	map(0x00000000, 0x0009ffff).ram();
+	map(0x000e0000, 0x000fffff).rom().region("bios", 0xe0000);
+	map(0xfff00000, 0xffffffff).rom().region("bios", 0);
+}
+
+void aristmk7_state::aristmk7_io(address_map &map)
+{
+}
+
+static INPUT_PORTS_START(aristmk7)
+INPUT_PORTS_END
+
+void aristmk7_state::aristmk7(machine_config &config)
+{
+	PENTIUM3(config, m_maincpu, 266'000'000 * 7); /* Celeron-M 440, 533 MHz FSB, 1.86 GHz, actual equation TBD */
+	m_maincpu->set_addrmap(AS_PROGRAM, &aristmk7_state::aristmk7_map);
+	m_maincpu->set_addrmap(AS_IO, &aristmk7_state::aristmk7_io);
+
+	PCI_ROOT(config, "pci", 0);
+	// ...
+}
+
+// Phoenix based BIOS
+#define ARISTMK7_BIOS \
+	ROM_REGION32_LE(0x100000, "bios", 0) \
+	ROM_LOAD( "mk7-bios_590176_01_1.00.7.u1", 0x000000, 0x100000, CRC(f2fea07e) SHA1(285038473fbe3a5708045e2f8b14562f3c24b203) )
+
+
+ROM_START( aristmk7 )
+	ARISTMK7_BIOS
+ROM_END
+
+ROM_START( a7lucky88 )
```

Comment:
I don't like this shortname, but `lucky88` is already taken by the MK-6 version ...